ui/css/itcss/components/newui-sections.scss

```diff
@@ -132,24 +108,6 @@
   }
 }
 
-.unlock-screen-container {
-  z-index: design-system.$main-container-z-index;
-  display: flex;
-  justify-content: center;
-  align-items: center;
-  flex: 1 0 auto;
-  background: var(--color-background-alternative);
-  width: 100%;
-}
-
-.unlock-screen {
-  display: flex;
-  flex-direction: column;
-  justify-content: center;
-  align-items: center;
-  flex: 1 0 auto;
-}
```

Comment:
Not used

![Screenshot 2025-02-07 at 11 56 11 AM](https://github.com/user-attachments/assets/e90bd6a8-ace9-4478-a0eb-0cd26a3ecfc8)